docs/meshstack.aks-developer-platform-quickstart-guide.md

```diff
@@ -0,0 +1,299 @@
+- You have integrated an AKS cluster into meshStack.
+    - *If you don’t, we recommend running through the [First Platform guide for AKS](./meshstack.aks-platform-quickstart-guide.md) where you are guided to integrate an AKS cluster as a platform in meshStack*
+
+## What will we do in this guide?
+
+Shortly put, we will set up the following in this guide:
+
+- A building block definition that allows application teams to provision new repositories.
+- A building block definition that adds a GitHub Action workflow with the right credentials in the AKS namespace so application teams have an out-of-the-box working Kubernetes CI/CD.
+- A building block definition that combines all these elements & more into a single packaged starter kit.
+
+## 1. GitHub Repository
+
+### Preparation
+
+The GitHub Repository building block definition needs access to your GitHub organization.
+Follow the following steps to create a GitHub App that will be used to manage the repositories.
+
+1. [**Register a GitHub App**](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app)
+    1. Make sure you give it sufficient permissions. When creating the app you’ll see a section “Repository permissions”. Under here make sure that you set the following:
+        1. “Administration” is set to “read & write”.
+        2. “Environments” is set to “read & write”.
+        3. “Secrets” is set to “read & write”.
+        4. “Contents” is set to “read & write”.
+    2. After creating the app you will see an “App ID” listed. Make sure to save this somewhere as we will need it later.
+2. Install the GitHub App
+    1. On the left-hand side go to “Install App” and install the app where you want to manage repositories.
+    2. Follow the guide and once finished you will end up on the installation configuration page. On this page, have a look at the URL and extract the number behind `/installations/<installation ID>`. Make sure to save this somewhere as we will need it later.
+3. [**Create a GitHub App Private Key**](https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/managing-private-keys-for-github-apps)
+    1. Save this file somewhere convenient as we will also need it later.
+
+### Creating the Building Block Definition
+
+To make things easy, we will use the [meshStack Hub](https://hub.meshcloud.io) to create the new building block definition.
+
+1. Open the Github Repository Creation Definition in the hub [here](https://hub.meshcloud.io/definitions/github-repository).
+2. Click on “Add to meshStack” and enter your meshStack URL (if not given already).
+3. Continue the flow in meshStack.
+    1. Right away you will be asked for a “Type”. Pick “Workspace Building Block” here.
+    2. Continue until you reach the “Inputs” page. On this page you will be forced to fill in the following:
```

Comment:
When this guide is available we should share it with Edeka. They are waiting eagerly to integrate with GitHub